cmd/plume/prerelease.go

```diff
@@ -404,7 +534,13 @@ func awsUploadToPartition(spec *channelSpec, part *awsPartitionSpec, imageName,
 		}
 
 		plog.Printf("Creating EBS snapshot...")
-		snapshot, err = api.CreateSnapshot(imageName, s3ObjectURL, aws.EC2ImageFormatVmdk)
+
+		format := aws.EC2ImageFormatVmdk
```

Comment:
Same comment as https://github.com/coreos/mantle/pull/934#discussion_r253616375

Don't instantiate the value and then have a special case for fedora, use an `if` or `switch` and set the value depending on the distro.